src/users/cd-login.vue

```diff
@@ -17,13 +36,52 @@
       return {
         email: '',
         password: '',
+        loginFailed: false,
       };
     },
     methods: {
+      async validateForm() {
+        try {
+          return await this.$validator.validateAll();
```

Comment:
I believe the try-catch is unecessary here : 
http://vee-validate.logaretm.com/examples.html#validate-form

Reply:
https://vee-validate.logaretm.com/api.html#validator-validate Docs says it can throw an error but it's not validation related. At that point, i'd prefer the error to bubble up

Reply:
Fair enough. We hit validator throwing an error before, but never really dug into why. It only occurred when something was invalid, so for us 1+1=2, but it obviously seems we were wrong. 😅